components/detail/SwapDetailSummary/SwapDetailAddress.tsx

```diff
@@ -18,6 +23,27 @@ function SwapDetailAddress(props: SwapDetailItem) {
         {wallet?.address}
       </Link>
       <ButtonCopyIcon tooltipText="Copy Address" text={wallet?.address} />
+      <button
+        className={`ml-1.5 flex items-center relative`}
+        onClick={() =>
```

Comment:
In order to make it easier to follow up with Google Analytics or other tools and to be able to open in a new tab, all `onClicks` have been refactored, and we used the link tag, please do this like the others.

like this in request id cell in table:

```
          <Link
            href={`/swap/${requestId}`}>
          </Link>
```

Reply:
fixed